.devcontainer/Dockerfile

```diff
@@ -20,42 +25,20 @@ RUN apt-get update && apt-get -y install postgresql-${PG_MAJOR}-postgis-3  \
                                          postgresql-client-${PG_MAJOR}     \
                                          libpq-dev
 
-# install azure-cli and azurite
```

Comment:
Local cloud storages minio, azurite, and fake-gcs-server are setup via docker-compose file. We run them via docker run at CI similarly.